lib/queue.js

```diff
@@ -61,6 +63,19 @@ var Queue = function Queue(name, redisPort, redisHost, redisOptions){
     redisOptions.db = redisOpts.DB;
   }
 
+  if (_.isString(redisPort)) {
```

Comment:
For more stable code I think it is better to use an else if here (i.e. if redisPort is not an Object, then check if it is a string.